### DISCUSSION TITLE
Youtube brightness seekbar 'auto' removal

### DISCUSSION THREAD

**Discussion by stickyburn** [ReVanced/revanced-discussions#981] [Question] [answered]
I loved vanced seekbar where I could easily swipe down for 0% brightness. Revanced has the same patch but now swiping down makes the brightness auto making it very tough to set 0%.

Do you think this is a feature worth enough to be requested? If people support I will add an issue.

**Comment by Novgorod** [comment]
Got any luck with that yet? I'm pretty sure early Revanced versions had switchable auto-brightness mode where swiping would go down all the way to 0% instead of the useless "auto" (0% doesn't exist anymore, only 1% which is very hard to hit).. Any mention of this issue in the Revanced subreddit gets your post immediately deleted, looks like a sensitive topic.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
There is now a swipe option to not enable auto brightness and instead use 0 brightness

**Comment by Xeldin** [comment]
> There is now a swipe option to not enable auto brightness and instead use 0 brightness 

Where? And which revanced version?

**Reply by oSumAtrIX** [reply]
In the swipe controls settings in the latest version.
